bigflow/cli.py

```diff
@@ -493,23 +468,26 @@ def _cli_deploy_image(args):
         vault_secret = _resolve_property(args, 'vault_secret')
     except ValueError:
         vault_secret = None
-    if args.image_tar_path:
-        build_ver = _decode_version_number_from_file_name(Path(args.image_tar_path))
-        image_id = load_image_from_tar(args.image_tar_path)
-        tag_image(image_id, docker_repository, build_ver)
-    else:
-        build_ver = args.version
+    image_tar_path = args.image_tar_path if args.image_tar_path  else _find_image_file()
 
-    deploy_docker_image(build_ver=build_ver,
+    deploy_docker_image(image_tar_path=image_tar_path,
                         auth_method=args.auth_method,
                         docker_repository=docker_repository,
                         vault_endpoint=_resolve_vault_endpoint(args),
                         vault_secret=vault_secret)
 
 
+def _find_image_file():
```

Comment:
In general, we don't use underscores. If you want to say which element from a module is public, use __all__.